### DISCUSSION TITLE
When the next pypi release?

### DISCUSSION THREAD

**Discussion by robert011192** [agronholm/anyio#775] [General]
Hi everyone,

I'm wondering when you plan to release the next PyPI version. I'm waiting to update my app, which depends on a fix made in this PR https://github.com/agronholm/anyio/pull/767. 

Cheers

**Comment by agronholm** [comment]
I will make a release once all these pending PRs have been reviewed and merged:

- https://github.com/agronholm/anyio/pull/750
- https://github.com/agronholm/anyio/pull/752
- https://github.com/agronholm/anyio/pull/761
- https://github.com/agronholm/anyio/pull/774
- https://github.com/agronholm/anyio/pull/777
- https://github.com/agronholm/anyio/pull/779

**Reply by agronholm** [reply]
I'm just awaiting reviews on two PRs now.

**Reply by agronholm** [reply]
Only one to go now, and that one's being reviewed. Best case scenario, there'll be a release tomorrow.

**Comment by agronholm** [comment]
I've released both v4.5.0 which is the last feature release with Python 3.8 support, and v4.6.0 which contains significant fixes to cancellation behavior on asyncio, but also drops Python 3.8 support.
